### PR TITLE
www-render and listening service integration phase 1a

### PR DESCRIPTION
**Pull request by toddwelstein**
Allow for the user's anonymous access token to be sent from www-render -> NPR One Backend Proxy -> Identity Service

### CODE REVIEW

File: src/Controllers/AuthCodeController.php [hunk 1]
```diff
@@ -111,6 +112,7 @@ public function startAuthorizationGrant(array $scopes, ?string $email=null): str
             'response_type' => 'code',
             'scope'         => join(' ', $scopes),
             'email'         => $email,
+            'user_id'       => $accessToken,
```
Comment (iancampbellstuff) [inline src/Controllers/AuthCodeController.php]: Should this query string parameter be conditionally added, if `$accessToken` is not null?

**Review by iancampbellstuff — APPROVED** [review]

File: src/Controllers/AuthCodeController.php (hunk 1 above)
Comment (iancampbellstuff) [reply src/Controllers/AuthCodeController.php]: Seeing these changes are used in https://github.com/npr/www-render/pull/1806

Comment (toddwelstein) [reply src/Controllers/AuthCodeController.php]: Yeah that makes sense. I was following the existing pattern of how "$email" is handled, though.

Comment (shannonphillipl) [reply src/Controllers/AuthCodeController.php]: Can this be an access token, and not an actual user ID?

Comment (toddwelstein) [reply src/Controllers/AuthCodeController.php]: @todo Jared to get back to me on this on how to best send access token, or instead to use user model

Comment (toddwelstein) [reply src/Controllers/AuthCodeController.php]: If there is code already in www-render that gets a user model from an anonymous Id, then can use that to send along here

**Pull request merged**